innopoints/core/notifications/__init__.py

```diff
@@ -33,8 +34,7 @@ def send_mail_async(message):
         mail_thread.start()
         log.info(f'Sent an email to {recipient_email}')
     elif channel == 'push':
-        # TODO: send Push
-        pass
+        push(recipient_email, notification_type, payload)
```

Comment:
Why'd you have to accept arguments in a different order)

Reply:
I just copied it from somewhere I don't remember 😁